src/pal/inc/unixasmmacrosx86.inc

```diff
@@ -35,3 +35,31 @@ C_FUNC(\Name\()_End):
         .global C_FUNC(\Name\()_End)
         LEAF_END \Name, \Section
 .endm
+
+.macro PROLOG_BEG
+        push    ebp
+        mov     ebp, esp
+        .cfi_def_cfa_offset     0
```

Comment:
This is not correct. The CFA by definition is the ESP of the caller right before it made the call. You are setting the CFA here to the location where the EBP was pushed. 
You need to set the offset to 8 (the pushed EBP plus the return address).